src/index.scss

```
@@ -237,3 +237,16 @@ $baseline-interactive-font-size: 17px;
 		min-width: 274px !important;
 	}
 }
+
+// iOS-specific styling for block settings menu
+body.is-ios.gutenberg-kit {
+	.block-settings-menu {
+		background-color: rgba(0, 0, 0, 0.3) !important;
+	}
+
+	.block-settings-menu .components-popover__content {
+		border-top-left-radius: 26px !important;
+		border-top-right-radius: 26px !important;
+		background-color: #ffffff !important;
+	}
+}
```

I feel we should:

- Avoid `!important` every whenever possible, it often creates future specificity conflicts. 
- Use the lowest level of selector specificity necessary to allow reconciling future, unpredictable styling needs. 
- If we are going to rely upon the Sass nesting syntax, leverage it to its full extent to avoid repetition. 
- Alphabetize attributes to simplify searching and modifying code. 

Feel free to share counter thoughts. 

```suggestion
.gutenberg-kit.is-ios .block-settings-menu {
	background-color: rgba(0, 0, 0, 0.3);

	& .components-popover__content {
		background-color: #ffffff;
		border-top-left-radius: 26px;
		border-top-right-radius: 26px;
	}
}
```